src/components/signup/SigninForm.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import fire from 'config/firebase';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { Form, Button } from 'semantic-ui-react';
+import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+
+import { closeModalAction } from 'store/actions/signupAction';
+import signInUserAction from 'store/actions/signInUserAction';
+import { relative } from 'upath';
+
+class SigninForm extends Component {
+    state = {
+        email: '',
+        password: '',
+        user: {}
+    };
+
+    uiConfig = {
+        signInFlow: 'popup',
+        signInOptions: [fire.auth.GoogleAuthProvider.PROVIDER_ID],
+        callbacks: {
+            signInSuccess: () => false
+        }
+    };
+
+    componentDidMount() {
+        this.authListener();
```

Comment:
Nitpicky too, but I would personally prefer the name of this method to be a tiny bit more descriptive.
What does `authListener` do? As someone who's seen the code for the first time, the first thought is that it would be a listener itself. But wait, we call it in `componentDidMount`? So I look below to see the code and I realize it's a method that attaches an event listener, not the event listener itself. Think

```javascript
// this is an event listener
handleClick = () => {
  doSomething();
}

// this is a method that attaches an event listener
const attachHandleClick = () => {
  $(".button").click(handleClick)
}
```

So I would change the name of this to `attachAuthListener`